src/components/Blockquote.tsx

```diff
@@ -17,91 +17,37 @@ interface Link {
     href: string
 }
 
-export const BlockquoteWithLogoBottom: FunctionComponent<{
```

Comment:
I thought this should combine with the generic `Blockquote` since that component allows for a Logo at the bottom right now anyway